build.gradle

```diff
@@ -103,6 +103,9 @@ rat {
             'node_modules/**',
             '.idea/**',
             'README.md',
+            'docker/Dockerfile',
+            'docker/docker-compose.yml',
+            'docker/assets/',
```

Comment:
Does it make sense to use something like docker gradle to do the build? https://github.com/Transmode/gradle-docker